enhancements/network/ovn-kubernetes-multiple-external-gateway.md

```diff
@@ -0,0 +1,180 @@
+        *`static`: is a slice of static IPs, each item containing the following fields:
+            * `ip`: IPv4 or IPv6 of the next destination hop. This field is mandatory.
+            * `skipHostSNAT`: When true, it disables applying the Source NAT to the host IP.
+            * `bfdEnabled`: determines if Bi-Directional Forwarding Detection (BFD) is supported in this network and follows the same behavior as with the annotation: it's enabled when set to true and disabled when false or omited in the manifest. It maps to the `k8s.ovn.org/bfd-enabled` annotation in the pod. This field is optional.
+        * `dynamic`: references contain the following fields:
+          * `podSelector` implements a [set-based](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement) label selector to filter the pods in the namespace that match this network configuration. These are dynamic IPs linked to the lifecycle of the pod. This field is mandatory.
+          * `namespaceSelector` It defines a `set-based` selector that is used to filter the namespaces where to apply the `podSelector`. This field is optional.
+          * `skipHostSNAT`: When true, it disables applying the Source NAT to the host IP.
+          * `bfdEnabled`: determines if Bi-Directional Forwarding Detection (BFD) is supported in this network and follows the same behavior as with the annotation: it's enabled when set to true and disabled when false or omited in the manifest. It maps to the `k8s.ovn.org/bfd-enabled` annotation in the pod. This field is optional.
+          * `networkAttachmentName`: Name of the network attached definition. The name needs to match from the list of logical networks generated by Multus and associated to the pod via annotation. This field is optional. When this field is empty or not defined, the IP from the pod's host network will be used instead. The pod must have `hostnetwork` enabled in order for this feature to work.
+
+The following yaml examples an instance of the CRD:
+
+```yaml
+apiVersion: k8s.ovn.org/v1
+kind: AdminPolicyBasedExternalRoute
+metadata:
+  name: honeypotting
+spec:
+  policies:
+  ## gateway example
+  - from:
+      namespaceSelector:
+        matchLabels:
+            multiple_gws: true
+    nexthops:
+      static:
+        - ip: "172.18.0.2"
+          bfdEnabled: true
+        - ip: "10.244.0.3"
+          skipHostSNAT: true
+      dynamic:
+        - podSelector:
+            matchLabels:
+              external-gateway: true
+          bfdEnabled: true
+          namespaceSelector:
+            matchLabels:
+              gateway: true
+          networkAttachmentName: sriov1
+        - podSelector:
+            matchLabels:
+              secondary-gateway: true
+          skipHostSNAT: true
```

Comment:
Is this optional field? Does that mean the value is going to be `false` by default?

Reply:
Correct. It's optional and defaults to `false`.